scripts/readme.md

```diff
@@ -0,0 +1,13 @@
+# Auto Issue Assign Instruction
+## Define Variables
+Make sure to define correct repository variables in repo setting page. Enter github username in `ONCALL_LIST`. Custom oncall info is in `CUSTOM_ONCALL_ROTATION` with the following format:
+
+`{github-username},{start-date},{end-date}`
+
+**Do not put space between commas!**
+
+Add new entries by starting a new line.
```

Comment:
here you mention that we should add new entries by starting a new line, but in the code it looks like new entries are determined through splitting the array by `;`? Do you mind fixing this readme to have the correct format, and also include an example of a multiple date entry?

`{github-username-1},{start-date},{end-date};{github-username-2},{start-date},{end-date}`

Also, I'm still unclear from this readme as to whether these dates are inclusive or exclusive, as i mentioned in https://github.com/microsoft/Oryx/pull/1902#discussion_r1124513804

Reply:
thanks! I missed that earlier on.